docs/reference/objects/company.md

```diff
@@ -4,60 +4,65 @@ title: Company
 parent: Objects
 ---
 
-Company is a _global_ variable, meaning it should always be accessible to you when developing on canvas
+Company is a _global_ variable, meaning it will always be accessible on any Liquid template.
 
-# company.products
-
-This will return an array of the companies published [products]({% link docs/reference/objects/product/index.md %}) excluding any private products
-e.g.
-
-{% raw %}
-```liquid
-{% for product in company.products %}
-    {{ product.name }}
-{% endfor %}
-```
-{% endraw %}
+# company.authors
 
+A list of all the company's [authors]({% link docs/reference/objects/author.md %}).
 
-# company.series
+# company.avatar_logo
 
-This will return an array of the companies [series]({% link docs/reference/objects/series.md %})
+The company's logo as an [image]({% link docs/reference/objects/image.md %}). The logo is set within **Settings** > **Your Brand**.
 
 # company.biographies
 
-This will return an array of the companies [biographies]({% link docs/reference/objects/biography.md %})
+A list of all the company's [biographies]({% link docs/reference/objects/biography.md %}).
 
 # company.countries
 
-This will return an array of the names of countries with products
-e.g.
+A list of the names of countries with products. e.g. `["Canada", "United Kingdom"]`
 
-# company.name
+# company.facebook_username
 
-Returns the name of the company
+The facebook username set within **My Site** > **Social**.
 
-# company.subdomain
+# company.instagram_username
 
-Returns the subdomain of the company
+The instagram username set within **My Site** > **Social**.
 
-# company.facebook_username
+# company.logo
 
-Returns the facebook username set within My Site > Social
+The company's workmark logo as an [image]({% link docs/reference/objects/image.md %}). The logo is set within **Settings** > **Your Brand**.
+# company.name
 
-# company.instagram_username
+The name of the company.
 
-Returns the instagram username set within My Site > Social
+# company.posts
 
-# company.twitter_username
+A list of all the company's published [blog posts]({% link docs/reference/objects/blog_post.md %}), ordered by the most recently published first.
 
-Returns the twitter username set within My Site > Social
+# company.products
 
-# company.avatar_logo
+A list of the company's published [products]({% link docs/reference/objects/product/index.md %}) excluding any private products.
+e.g.
+
+{% raw %}
+```liquid
+{% for product in company.products %}
+    {{ product.name }}
+{% endfor %}
+```
+{% endraw %}
 
-Returns the company's logo as an [image]({% link docs/reference/objects/image.md %}). The logo is set within Settings > Your Brand.
 
-# company.logo
+# company.series
 
-Returns the company's workmark logo as an [image]({% link docs/reference/objects/image.md %}). The logo is set within Settings > Your Brand.
+A list of all the company's [series]({% link docs/reference/objects/series.md %}).
```

Comment:
In each of these cases where the return is a list of drops I'd state that explicitly. It is implied given you don't say series ids or series names but having it explicit preferable

Reply:
Do you not think it's obvious enough that it links to the documentation for the object? That's the pattern that we mostly use throughout these docs. We can start being more explicit though if you think it's needed. However, I feel like it's the exception to the rule to return IDs or names and we would state that in the method if so.

Reply:
oo yes actually you're right, the styling in the docs makes this very obvious.